docker-compose.yml

```diff
@@ -49,6 +50,24 @@ x-honeycomb-variables: &honeycomb-variables
 
 version: '3.4'
 services:
+  elasticsearch:
```

Comment:
Should we update our deploy scripts to add ES?

Reply:
You did not set it as a dependency of web... so it wont be started.

Reply:
This is for local development only... it does get started when do docker-compose up locally. For staging/prod, you just have to do ‘make apply’ in idseq-infra

Reply:
This is for local development only... it does get started when do docker-compose up locally. For staging/prod, you just have to do ‘make apply’ in idseq-infra

Reply:
Can you add the dependency? 
I only do `docker-compose up web` because most of the time i do not need the rescue workers. So it would be nice to be able to keep up doing that :)

Reply:
makes sense. I added it :)